patches/minecraft/net/minecraft/world/entity/item/ItemEntity.java.patch

```diff
@@ -32,7 +32,7 @@
           Vec3 vec3 = this.m_20184_();
           float f = this.m_20192_() - 0.11111111F;
 +         net.minecraftforge.fluids.FluidType fluidType = this.getMaxHeightFluidType();
-+         if (!fluidType.isAir() && !fluidType.isVanilla() && this.getFluidTypeHeight(fluidType) > (double)f) fluidType.setItemMovement(this);
++         if (!fluidType.isAir() && !fluidType.isVanilla() && this.getFluidTypeHeight(fluidType) > (double)f) fluidType.setItemMovement(this); else
```

Comment:
Should we consider expanding this into two additional lines, so the detail of the `else` can't be glossed over?

Reply:
Yes we should.